### PR TITLE
Contributor meeting sponsor changes and other feedback

### PR DESCRIPTION
**Pull request by Aggror**
Made some contributor display changes and small other feedback on community meeting post

### CODE REVIEW

File: posts/2023-10-07-community-meeting-october-2023.md [hunk 1]
```diff
@@ -97,17 +89,33 @@ Members have been asking if they can help out with the rewrite, but up until thi
...
 ### Platinum Striders
+* <div>
 
-* {% include sponsor-org.md key:'happenstance' emoji:'🏆' %}
+  [Happenstance Games](www.happenstancegames.com) 
+
+  <a href="www.happenstancegames.com/"><img width=64 src="https://www.stride3d.net/images/sponsors/HappenLogo.webp" alt="vvvv. A visual live-programming environment that takes you from rapid prototyping to final production"></a>
+</div>
 
 ### Gold Striders
```
Comment (VaclavElias) [inline posts/2023-10-07-community-meeting-october-2023.md:113]: Can we add these to to your PR?

File: posts/2023-10-07-community-meeting-october-2023.md [hunk 2]
```diff
@@ -97,17 +89,33 @@ Members have been asking if they can help out with the rewrite, but up until thi
...
+  <a href="https://visualprogramming.net/"><img src="https://images.opencollective.com/vvvv/8ab0acd/logo/256.png?height=72" alt="vvvv. A visual live-programming environment that takes you from rapid prototyping to final production"></a>
+</div>
+
 ### Platinum Striders
+* <div>
 
-* {% include sponsor-org.md key:'happenstance' emoji:'🏆' %}
+  [Happenstance Games](www.happenstancegames.com) 
+
+  <a href="www.happenstancegames.com/"><img width=64 src="https://www.stride3d.net/images/sponsors/HappenLogo.webp" alt="vvvv. A visual live-programming environment that takes you from rapid prototyping to final production"></a>
```
Comment (Eideren) [inline posts/2023-10-07-community-meeting-october-2023.md:110]: alt text is wrong

Comment (Aggror) [reply posts/2023-10-07-community-meeting-october-2023.md:110]: good catch

File: posts/2023-10-07-community-meeting-october-2023.md (hunk 1 above)
Comment (Aggror) [reply posts/2023-10-07-community-meeting-october-2023.md:113]: Sorry? I don't understand

Comment (VaclavElias) [reply posts/2023-10-07-community-meeting-october-2023.md:113]: Resolved :)

**Pull request merged**